airbrakes/airbrakes.py

```diff
@@ -101,14 +101,15 @@ def update(self) -> None:
 
         # Update the processed data with the new data packets. We only care about EstimatedDataPackets
         self.data_processor.update(est_data_packets)
-        if self.state.name[0] == "C":  # Only run apogee prediction in coast state:
-            pass
-            # self.apogee_predictor.update(est_data_packets)
 
         # Get the processed data packets from the data processor, this will have the same length as the number of
         # EstimatedDataPackets in data_packets
         processed_data_packets: deque[ProcessedDataPacket] = self.data_processor.get_processed_data_packets()
 
+        if isinstance(self.state, CoastState):  # Only run apogee prediction in coast state:
```

Comment:
I deliberately didn't use `isinstance` for performance reasons. Not pythonic, but just pre-optimizing based on previous profiling. I'll specifically profile line by line later.

Reply:
okay I'll revert.

Reply:
does it actually matter thaaat much though? If so, we should see If we can optimize this further
```py
        est_data_packets = [
            data_packet
            for data_packet in imu_data_packets.copy()
            if isinstance(data_packet, EstimatedDataPacket)
            # The copy() above is critical to ensure the data here is not modified by the data processor
        ]
```